letsencrypt-auto-source/pieces/bootstrappers/rpm_common.sh

```diff
@@ -25,9 +25,9 @@ BootstrapRpmCommon() {
   fi
 
   if ! $SUDO $tool list *virtualenv >/dev/null 2>&1; then
-    echo "To use Certbot, packages from the EPEL repository need to be installed."
+    say "To use Certbot, packages from the EPEL repository need to be installed."
```

Comment:
I think we could probably try running `$tool list ...`, capture and process the output and then decide to spit out the message and run `sudo $tool ...`.